Cataclysm/Rogue.lua

```diff
@@ -1092,7 +1098,8 @@ spec:RegisterAbilities( {
     },
 
 
-    --[[ Enrages you, increasing all damage caused by 5%.  Requires a bleed effect to be active on the target.  Lasts 1 min.
+    -- Enrages you, increasing all damage caused by 5%.  Requires a bleed effect to be active on the target.  Lasts 1 min.
```

Comment:
Show me Hunger for Blood in-game?  Spell ID 63848 doesn't exist on Wowhead.

Reply:
Aaah then we might aswell take it away. Eatherway it was busting kick